docs/hardware_setup.md

```diff
@@ -49,9 +49,9 @@ It is important to remember that when launching the VIO, the camera should be st
 
 3. Calibrate camera intrinsics and extrinsics using [Kalibr](https://github.com/ethz-asl/kalibr), recommended model is: ```pinhole-equi``` [(see OpenCV documentation)](https://docs.opencv.org/3.3.1/db/d58/group__calib3d__fisheye.html)
 
-4. Create configuration files for KimeraVIO ROS wrapper using [Kalibr2KimeraVIO-pinhole-equi](https://github.mit.edu/SPARK/VIO/blob/feature/parallelization/jpl/kalibr/kalibr2sparkvio_stereo_pinhole-equi.py) or [[Kalibr2KimeraVIO-pinhole-radtan](https://github.mit.edu/SPARK/VIO/blob/feature/parallelization/jpl/kalibr/kalibr2sparkvio_stereo_pinhole-equi.py)]
+4. Create configuration files for KimeraVIO ROS wrapper using [Kalibr2KimeraVIO](https://github.com/MIT-SPARK/Kimera-VIO/kalibr2kimeravio.py)
```

Comment:
This one also gives me 404 (but I can see it here: https://github.com/MIT-SPARK/Kimera-VIO/blob/fix/names_kalibr_files/config2kimeravio.py)